src/prefs/TracksBehaviorsPrefs.cpp

```diff
@@ -112,10 +112,6 @@ void TracksBehaviorsPrefs::PopulateOrExchange(ShuttleGui & S)
       S.TieCheckBox(XXO("Use dialog for the &name of a new label"),
                     {wxT("/GUI/DialogForNameNewLabel"),
                      false});
-#ifdef EXPERIMENTAL_SCROLLING_LIMITS
```

Comment:
See `SCROLLING_LIMITS` in `src/Experimental.cmake`.  You delete the sole use of `EXPERIMENTAL_SCROLLING_LIMITS`, and that is where this preprocessor constant is defined.

See my comments there about why I implemented scrolling beyond zero:

```
   # Paul Licameli (PRL) 24 May 2015
   # Allow scrolling up to one half of a screenful beyond either end of the project,
   # if you turn on the appropriate Tracks preference.
   # This allows smooth-scrolling scrub to work more reasonably at the ends.
```

I'd forgotten that motivation, myself!

Does this give any reason to pause about deleting this feature?

I had certain ideas about scrubbing -- I don't remember all of them just now, but I think one of them was an alternative mode that would keep the play head in a fixed position but let you drag the wave under it.

Reply:
If not, then also delete this part of Experimental.cmake